docs/source/x_ext.rst

```diff
@@ -196,7 +203,9 @@ A SystemVerilog interface implementation for CORE-V-XIF could look as follows:
     parameter int          X_RFR_WIDTH     =  32, // Register file read access width for the eXtension interface
     parameter int          X_RFW_WIDTH     =  32, // Register file write access width for the eXtension interface
     parameter logic [31:0] X_MISA          =  '0, // MISA extensions implemented on the eXtension interface
-    parameter logic [ 1:0] X_ECS_XS        =  '0  // Default value for ``mstatus.xs``
+    parameter logic [ 1:0] X_ECS_XS        =  '0, // Default value for ``mstatus.xs``
+    parameter logic        X_DUALREAD      =  0,  // Dual register file read
```

Comment:
If legal values are `int (0..3)`, shouldn't this be wider?

Reply:
Thanks. This should be fixed now.